src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -26,13 +27,13 @@ public interface MachineService {
      * @param machines - list of machines to be filled with value
      * @param value    - any object of machine sub class
      */
-    void fill(List<Object> machines, Object value);
+    void fill(List<? super Machine> machines, Machine value);
```

Comment:
The `fill` method should use the generic type `T` instead of `Machine` for the `value` parameter. This will allow the method to work with any subclass of `Machine` as intended by the interface's parameterization.